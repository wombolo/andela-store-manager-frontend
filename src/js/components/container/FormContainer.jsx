import React, { Component } from "react";

import Input from "../presentational/Input.jsx";

class FormContainer extends Component {
  state = {seo_title: ""};

  handleChange = (event) => {
    this.setState({ [event.target.id]: event.target.value });
  };

  render() {
    const { seo_title } = this.state;

    return (
      <form id="article-form">
        <Input
          text="SEO title"
          label="seo_title"
          type="text"
          id="seo_title"
          value={seo_title}
          handleChange={this.handleChange}
        />
      </form>
    );
  }
}

export default FormContainer;
