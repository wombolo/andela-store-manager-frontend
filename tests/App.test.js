import React from 'react';
import ReactDOM from 'react-dom';
import FormContainer from "../src/js/containers/FormContainer.jsx";


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<FormContainer />, div);
  ReactDOM.unmountComponentAtNode(div);
});
