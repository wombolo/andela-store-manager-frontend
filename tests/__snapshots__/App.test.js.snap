// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders without crashing 1`] = `
<BrowserRouter>
  <div
    className="row"
  >
    <div
      className="col-2 no-margin no-padding"
    >
      <Connect(Menu) />
    </div>
    <div
      className="col-8 min-height-100vh"
    >
      <Connect(TopBar) />
      <Routes />
    </div>
  </div>
</BrowserRouter>
`;
